software_bundle/README.md

```diff
@@ -0,0 +1,219 @@
+### Software bundle management
+
+#### What is a software bundle?
+A software bundle is a single file (e.g. RPM, squashfs, zip) which can be installed
+on the system to extends the functionality of the system in some way.   
+A software bundle may be installed to:
+- fix some bug/CVE on the system, by updating the programs affected by that
+  bug/CVE
+- add a new feature to the system
+- add/update some software utility (e.g. perl, ncurses)
+- update some dependency (e.g. updating some python site-package)
+
```

Comment:
The README should clarify if multiple versions of a software bundle may exist on the system (presumably only one is activated at any time).  Install/Uninstall need to be updated to handle picking which version is being used.

Reply:
done in https://github.com/openconfig/gnoi/pull/160/commits/4df2a72eaa307b8fbead852292a1f1293b34491a and included the suggestion that operators append the version into the name.